samples/cellular/nrf_cloud_multi_service/src/shadow_support_coap.c

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/logging/log_ctrl.h>
+#include <modem/location.h>
+#include <date_time.h>
+#include <stdio.h>
+#include <net/nrf_cloud.h>
+#include <net/nrf_cloud_coap.h>
+
+#include "shadow_support_coap.h"
+
+LOG_MODULE_REGISTER(shadow_support_coap, CONFIG_MULTI_SERVICE_LOG_LEVEL);
+
+#define COAP_SHADOW_MAX_SIZE 512
+
+static int check_shadow(void)
+{
+	int err;
+	char buf[COAP_SHADOW_MAX_SIZE];
+
+	buf[0] = '\0';
+	LOG_DBG("Checking for shadow delta...");
+	err = nrf_cloud_coap_shadow_get(buf, sizeof(buf), true);
+	if (err == -EACCES) {
+		LOG_DBG("Not connected yet.");
+	} else if (err) {
+		LOG_ERR("Failed to request shadow delta: %d", err);
+	} else {
+		size_t len = strlen(buf);
+
+		LOG_INF("Delta: len:%zd, %s", len, len ? buf : "None");
```

Comment:
`I want to keep the inf for the delta itself, if one is received.`
should this log message be moved below into the `if (len) {` block (and the "None" removed)?

not a big deal in its current location... it will print every 5 minutes and let users know the thread is still running.